standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -562,14 +562,27 @@ struct DateColumnStatsData {
 5: optional binary bitVectors
 }
 
+struct Timestamp {
+1: required i64 secondsSinceEpoch
```

Comment:
I do not think it is too complicated but it will imply changes in the metastore tables that store these values too. I did not want to change the internal representation of column stats for timestamp type in this patch, that is why I introduced the type but did not change the internal representation based on seconds. I created https://issues.apache.org/jira/browse/HIVE-22309.